### PR TITLE
Fixing selecting two rows in list view

### PR DESCRIPTION
**Pull request by ScopeyNZ**
This PR fixes the UI when choosing to select two versions to compare from the list screen.

I also made the actions column always render, because is was causing some awkward jittering as things became visible.

### CODE REVIEW

File: client/src/components/HistoryViewer/HistoryViewer.scss [hunk 1]
```diff
@@ -22,11 +22,10 @@
...
 }
 
-.history-viewer__table--current {
+.history-viewer__row--current {
   // There will only be one table row in this table state
-  tbody tr,
-  tbody tr:hover {
-    background-color: $link-color;
+  &, &:hover {
+    background-color: $link-color !important;
```
Comment (robbieaverill) [inline client/src/components/HistoryViewer/HistoryViewer.scss]: Please don't use `!important` - make the CSS more specific instead. If it's necessary in order to override another `!important` higher up can you add a comment to say that?

File: client/src/components/HistoryViewer/HistoryViewerVersionList.js [hunk 2]
```diff
@@ -82,6 +94,8 @@ HistoryViewerVersionList.propTypes = {
   messages: PropTypes.arrayOf(messageType),
   VersionComponent: PropTypes.oneOfType([PropTypes.node, PropTypes.func]).isRequired,
   versions: PropTypes.arrayOf(versionType),
+  compareFrom: versionType,
+  compareTo: versionType,
```
Comment (robbieaverill) [inline client/src/components/HistoryViewer/HistoryViewerVersionList.js]: You mentioned in Slack you were going to update this to an object, would be cool to do now before we merge more individual props

Comment (ScopeyNZ) [reply client/src/components/HistoryViewer/HistoryViewerVersionList.js]: I was hoping we'd get this in and then I'll touch it in the PR to fix up the object. Then I can add a prop type similar to how version currently is... It's a bit awkward trying to implement it in multiple PRs.

File: client/src/components/HistoryViewer/HistoryViewer.scss (hunk 1 above)
Comment (ScopeyNZ) [reply client/src/components/HistoryViewer/HistoryViewer.scss]: Yeah I felt bad when I did this... I'll be more specific, but I'll have to go right back to `table tbody tr.blah`.

Comment (robbieaverill) [reply client/src/components/HistoryViewer/HistoryViewer.scss]: This is why we need BEM everywhere!

Comment (ScopeyNZ) [reply client/src/components/HistoryViewer/HistoryViewer.scss]: Ok done. Wasn't as bad as I thought.

**Pull request merged**